Gem/Code/Source/Platform/Android/EntityLatticeTestComponent_Traits_Platform.h

```diff
@@ -12,3 +12,8 @@
 #define ENTITY_LATTICE_TEST_COMPONENT_HEIGHT                    2
 #define ENTITY_LATTICE_TEST_COMPONENT_DEPTH                     2
 #define ENTITY_LATTEST_TEST_COMPONENT_MAX                       4
+#define ENTITY_LATTICE_TEST_COMPONENT_SPACING_X                 5.0f
+#define ENTITY_LATTICE_TEST_COMPONENT_SPACING_Y                 5.0f
+#define ENTITY_LATTICE_TEST_COMPONENT_SPACING_Z                 5.0f
+#define ENTITY_LATTEST_TEST_COMPONENT_SPACING_MAX               100.0f
+#define ENTITY_LATTEST_TEST_COMPONENT_ENTITY_SCALE_MAX          10.0f
```

Comment:
Spacing and scaling don't really matter for platforms. Is there any concern that you added those?

Reply:
In this case I was following the lattice size example. I can remove the spacing and scale Pal-ification if necessary.